sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -408,6 +408,19 @@ trait GpuOverridesListener {
       costOptimizations: Seq[Optimization])
 }
 
+/**
+ * trait to check if the zone id is in UTC
+ */
+trait TimeZoneCheck {
+  def checkTimeZoneId(timeZoneId: Option[String], meta: RapidsMeta[_, _, _]): Unit = {
```

Comment:
This seems more appropriate as a static function than a trait.  We aren't going to use the trait class as a discriminator, and the trait doesn't leverage anything the class provides.